test/resque_test.rb

```diff
@@ -27,9 +27,9 @@
     new_redis = Redis.new(:host => "localhost", :port => 9736)
     new_namespace = Redis::Namespace.new("namespace", :redis => new_redis)
     Resque.redis = new_namespace
-    assert_equal new_namespace, Resque.redis
 
-    Resque.redis = 'localhost:9736/namespace'
```

Comment:
I think this was added to test this case, but we should only test one thing per test and there is a "can set a namespace through a url-like string" test above that tests this case